src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -90,8 +90,20 @@ def _delay(self):
 
     def __call__(self, poller):
         logger.info("Starting long running operation '%s'", self.start_msg)
+        correlation_message = ''
         while not poller.done():
-            self._delay()
+            try:
+                correlation_message = 'Correlation ID: {}' \
+                    .format(json.loads(poller._response.__dict__['_content']) #pylint: disable=protected-access
+                            ['properties']['correlationId'])
```

Comment:
>['properties']['correlationId']) [](start = 28, length = 32)

Is it always called correlationId? In the issue is says "operationId, correlationId,etc."